dance/modules/single_modality/imputation/deepimpute.py

```diff
@@ -290,7 +291,8 @@ def fit(self, X, Y, mask=None, batch_size=64, lr=1e-3, n_epochs=100, patience=5,
 
                 val_losses.append(val_loss)
                 min_val = min(val_losses)
-                if val_loss == min_val:
+                if val_loss == min_val or model_to_init:  #nan problem
```

Comment:
Even nan needs to be saved when the model is output for the first time

Reply:
Because all models need to be saved, otherwise the algorithm will report an error. When the model output is always nan, the model cannot be saved.